apps/yapms/src/lib/components/sidebar/SideBar.svelte

```diff
@@ -5,23 +5,25 @@
 	import { faReddit, faTwitter } from '@fortawesome/free-brands-svg-icons';
 
 	import titles from '$lib/assets/other/Titles.json';
+	import { SideBarStore } from '$lib/stores/SideBar';
 
 	$: title = titles.find((elem) => elem.path === $page.url.pathname)?.title ?? 'YAPms';
 </script>
 
-<div class="divider divider-horizontal w-0 m-0 hidden md:flex" />
-
-<div class="basis-3/12 max-w-md hidden md:inline">
-	<div class="flex flex-wrap justify-center gap-2 p-2">
-		<button type="button" class="btn btn-sm gap-2">
-			<Fa icon={faReddit} />
-			Reddit
-		</button>
-		<button type="button" class="btn btn-sm gap-2">
-			<Fa icon={faTwitter} />
-			Twitter
-		</button>
+{#if $SideBarStore}
+	<div class="divider divider-horizontal w-0 m-0 hidden md:flex" />
```

Comment:
`hidden md:flex` here means that the side bar can't be opened on mobile, making the button confusing. The way I see it, the best way to do that right now is to hide the button on mobile until we can rework the side bar to work on mobile.

Reply:
Added the same properties on the button, so now it will be hidden.